core/src/test/resources/tidb_config_alter_primary_key_true.properties.template

```diff
@@ -0,0 +1,33 @@
+# TiDB address
```

Comment:
From my perspective, it's a duplicate file?
There is no difference between this file and `tidb_config.properties.template`